regexMatches.js

```diff
@@ -288,11 +288,15 @@ module.exports = [
     },
   },
   {
-    regex: /#(\d{4})\b/,
+    // If contains either only PR references or some Git Hub related words
+    regex: /^(#\d{1,5})(\s+#\d{1,5})*$|\b(prs?|git(\s?hub)?|pull(\s?request)?s?|gh|issues?|commits?|merges?)\b/i,
     execute: (message, client) => {
-      const PRs = message.content.match(/#(\d{4})\b/g);
-      
-      message.reply(`Linked PRs detected\n\n${PRs.map(pr => `[PokéClicker PR${pr}](https://github.com/pokeclicker/pokeclicker/pull/${pr.replace('#', '')})`).join('\n')}`);
+      const PRs = message.content.match(/#(\d{1,5})\b/g);
+      // Ensure it matches at least once
+      if (!PRs.length) {
+        return;
+      }
+      message.reply(`Linked PRs detected : ${PRs.map(pr => `[PokéClicker PR${pr}](https://github.com/pokeclicker/pokeclicker/pull/${pr.slice(1)})`).join(', ')}`);
```

Comment:
Need these on new lines to show multiple PR embeds if there is more than one
```suggestion
      message.reply(`Linked PRs detected:\n${PRs.map(pr => `[PokéClicker PR${pr}](https://github.com/pokeclicker/pokeclicker/pull/${pr.slice(1)})`).join('\n')}`);
```

Reply:
Oh, wait, I am dumb, I did some testings and it seems to be caused by the comma I put... Did not know such thing would break Discord link finder.
![image](https://github.com/RedSparr0w/Discord-bot-pokeclicker/assets/68825215/2460fca3-eb89-45c1-a891-c28a0676bc4e)

Reply:
![image](https://github.com/RedSparr0w/Discord-bot-pokeclicker/assets/68825215/a4fbd318-2ee9-45e8-942c-977495dc1a57)
It works like that. I am pushing this. I will change it back if you prefer how it looked before.